examples/simple_listener/simple_listener.c

```diff
@@ -165,26 +197,78 @@ void sigint_handler(int signum)
 #endif /* PCAP */
 
 #if LIBSND
-	sf_write_sync(glob_snd_file);
-	sf_close(glob_snd_file);
+  if (NULL != glob_snd_file)
```

Comment:
Thank you for a great feedback. The reason why I propose that change is that my company is working on AVB Automotive stack and we have clients that are interested in using it with Open AVB. I have recently created  patch that allow to use Simple Listener for Automotive and I wanted to share it as maybe more people may be interested in it. However this avtp_pipeline listener may be a thing to go without all those changes (need to check).

Regarding your concerns:

Ad 1. Maybe you are right that I have made it too complex for example, I can limit the number of params to just 'a' for automotive and remove stream index

Ad 2. I agree that Automotive is a specific use-case but I believe that is worth to have it as an option for simple listener 

Ad 3. As much as I like to I can't currently work on development of matching simple_talker, but I believe that eventually I or someone else will update the talker with automotive option.